src/ui-test/tests/dev_mode_test.ts

```diff
@@ -0,0 +1,129 @@
+
+	describe(`Dev Mode test with the integration file update`, function () {
+		this.timeout(60000);
+
+		let driver: WebDriver;
+
+		const [initialCodeMessage, updatedCodeMessage, initialLogMessage, updatedLogMessage] = consts.prepareCodeLogMessages(extension, language);
+
+		before(async function () {
+			driver = VSBrowser.instance.driver;
+		});
+
+		after(async function () {
+			this.timeout(60000);
+			await new EditorView().closeAllEditors();
+			const item = await findSectionItem(consts.extensionName, consts.integrationFileName.toLowerCase());
+			if (item !== undefined) {
+				await driver.wait(() => { return contextMenuItemClick(item, consts.integrationRemove); });
+				const content = new SideBarView().getContent();
+				await driver.wait(async () => {
+					return viewHasItem(content, consts.extensionName, consts.integrationFileName.toLowerCase())
+						.then(val => {
+							return !val
+						});
+				});
+			}
+			await cleanOutputView();
+		});
+
+		beforeEach(function () {
+			if (!doNextTest.doNextTest) {
+				this.skip();
+			}
+		});
+
+		afterEach(function () {
+			if (this.currentTest?.state === 'failed' && this.id !== 'independent') {
+				doNextTest.stopTest();
+			}
+		});
+
+		it(`Select ${consts.startIntegration} in the popup menu`, async function () {
+			this.timeout(30000);
+			const item = await findSectionItem(consts.testFolder, `${consts.integrationFileName}.${extension}`);
+			await driver.wait(() => { return contextMenuItemClick(item, consts.startIntegration); });
+		});
+
+		it(`Start integration with '${devModeIntegration}' command`, async function () {
+			this.timeout(5000);
+			assert.isTrue(await inputBoxQuickPickOrSet('pick', devModeIntegration));
+		});
+
+		it(`Initial integration pod started`, async function () {
+			this.timeout(300000);
+			await driver.wait(() => { return outputViewHasText(consts.initialPodReadyMessage); });
+		});
+
+		it(`Initial integration output contains - ${initialLogMessage}`, async function () {
+			this.timeout(30000);
+			await driver.wait(() => { return outputViewHasText(initialLogMessage); });
+		});
+
+		it(`Integration output does not contain ASCI`, async function () {
+			this.timeout(30000);
+			this.id = 'independent'
+			assert.isTrue(await textDoesNotContainAsci('OutputView'));
+		});
+
+		it(`Integration exists in ${consts.extensionName} sidebar`, async function () {
+			this.timeout(30000);
+			const content = new SideBarView().getContent();
+			await driver.wait(() => { return viewHasItem(content, consts.extensionName, consts.integrationFileName.toLowerCase()); });
+		});
+
+		it(`Update Simple.${extension} with new message`, async function () {
+			this.timeout(30000);
+			await findSectionItem(consts.testFolder, `${consts.integrationFileName}.${extension}`);
+			await driver.wait(() => { return updateFileText(initialCodeMessage, updatedCodeMessage); });
+		});
+
+		it(`Updated integration pod started`, async function () {
+			this.timeout(300000);
```

Comment:
300000 ? more than 83 hours? OI guess you meant 30000, so 30 seconds

Reply:
right, I forgot to delete it when i realized it as it was in several places :-(

Reply:
Duplicated with next review :-)